sources/api/migration/migrations/v1.11.0/kubelet-aws-config/src/main.rs

```diff
@@ -0,0 +1,29 @@
+#![deny(rust_2018_idioms)]
+
+use migration_helpers::common_migrations::AddPrefixesMigration;
+use migration_helpers::{migrate, Result};
+use std::process;
+
+/// We added a new setting for configuring kubelet's image credential
+/// provider plugins. Initially this is only to support ecr-credential-provider,
+/// but others may be added as needed.
+fn run() -> Result<()> {
+    migrate(AddPrefixesMigration(vec![
+        "settings.aws.credentials",
+        "settings.aws.config",
+        "settings.aws.profile",
```

Comment:
These three settings should be covered by the `aws-creds` `AddSettings` migration, so they don't need to be listed here again